mods/gen2/moves.js

```diff
@@ -12,13 +12,17 @@ exports.BattleMovedex = {
 			if (target.boosts.atk >= 6) {
 				return false;
 			}
+
+			var boost = 0;
+
 			if (target.hp <= target.maxhp / 2) {
-				this.boost({atk: 2});
-				return false;
+				boost = (target.boost ? target.boost.atk : 0) + 2;
```

Comment:
I'm specifying it because ``this.boost({atk: 6}, target)`` will boost attack by six stages rather than increase it to 6, so that doesn't solve the problem (unless I'm wrong on that).